pkg/pillar/cmd/baseosmgr/handleverifier.go

```diff
@@ -50,17 +51,24 @@ func lookupVerifierConfig(ctx *baseOsMgrContext, objType string,
 }
 
 // If checkCerts is set this can return an error. Otherwise not.
-func createVerifierConfig(ctx *baseOsMgrContext, objType string, safename string,
-	sc *types.StorageConfig, checkCerts bool) error {
+func createVerifierConfig(ctx *baseOsMgrContext, uuidStr string, objType string,
+	safename string, sc *types.StorageConfig, ss *types.StorageStatus, checkCerts bool) error {
```

Comment:
Since this isn't supposed to modify StorageStatus, please define it as not being a pointer i.e. ss types.StorageStatus.